pilot/pkg/networking/core/v1alpha3/cluster_waypoint.go

```diff
@@ -60,7 +60,7 @@ func (configgen *ConfigGeneratorImpl) buildWaypointInboundClusters(
 	svcs map[host.Name]*model.Service,
 ) []*cluster.Cluster {
 	clusters := make([]*cluster.Cluster, 0)
-	// Creates "internal" cluster to route to the main "internal" listener.
+	// Creates "internal" cluster to route to the main listener.
```

Comment:
main does not equals `internal` anymore? The const InternalName = "internal"

Reply:
Changed the name to main_internal